lib/react_client.dart

```diff
@@ -919,6 +928,15 @@ _convertBoundValues(Map args) {
   }
 }
 
+void _convertRefValue(Map args) {
+  var ref = args['ref'];
+  if (ref != null) {
+    if (ref is react_interop.Ref) {
```

Comment:
#nit In Dart, `is`-checks on `null` are always false except for `is Null` or `is Object`, so you don't need an additional type check here.
```dart
void main() {
  print(null is int);      // false
  print(null is Function); // false
  print(null is MyClass);  // false
  print(null is Object);   // true
  print(null is Null);     // true
}

class MyClass {}
```